### PR TITLE
Add Windows installer

### PR DESCRIPTION
**Pull request by theSoenke**
Build Windows Installer with Innosetup

### CODE REVIEW

File: build/innosetup/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+
+RUN dpkg --add-architecture i386
+RUN apt-get update && apt-get install -y --no-install-recommends curl ca-certificates wine-stable wine32 xvfb
+
+ENV WINEDEBUG fixme-all
+ENV WINEARCH win32
+
+# # Install Inno Setup binaries
+RUN curl -SL "http://www.xs4all.nl/~mlaan2/ispack/innosetup-5.5.9.exe" -o is.exe
```
Comment (phraseapp-vincent) [inline build/innosetup/Dockerfile]: Please use official URL with link following and NOT direct mirror!
For example: `curl -SL "https://github.com/jrsoftware/issrc/releases/download/is-5_6_1/innosetup-5.6.1.exe" -o is.exe` for GitHub Release

File: build/innosetup/phraseapp-client.iss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{DC598D8E-8A9B-4CAD-AFD8-0324FDF4E0F1}
+AppName=PhraseApp Client
+AppVersion=1.7.3
+;AppVerName=PhraseApp Client 1.7.3
```
Comment (phraseapp-vincent) [inline build/innosetup/phraseapp-client.iss]: Why not uncomment the AppVerName?
Leave blank instead!

File: build/innosetup/phraseapp-client.iss [hunk 3]
```diff
@@ -0,0 +1,40 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{DC598D8E-8A9B-4CAD-AFD8-0324FDF4E0F1}
+AppName=PhraseApp Client
+AppVersion=1.7.3
```
Comment (phraseapp-vincent) [inline build/innosetup/phraseapp-client.iss]: the .version file does not match the AppVersion.
Which is right?

**Comment by theSoenke** [comment]
@phraseapp-vincent Thanks for your feedback. Should be all fixed now

**Pull request merged**